xml/System.Globalization/CultureInfo.xml

```diff
@@ -312,52 +312,10 @@
-  
-```csharp  
-  
-ResourceContext context = ResourceManager.Current.DefaultContext;  
-var list = new List<String>();  
-if (CultureInfo.CurrentCulture.Name == "en-US")  
-   list.Add("fr-FR");  
-else  
-   list.Add("en-US");  
-  
-context.Languages = list;  
-block.Text += String.Format("\nThe current culture is {0}\n",  
-                            CultureInfo.CurrentCulture);  
-block.Text += String.Format("The current culture UI is {0}\n",  
-                            CultureInfo.CurrentUICulture);  
-// The example displays output like the following if run on a system  
-// whose current culture is English (United States):  
-// The current culture is fr-FR  
-// The current UI culture is fr-FR  
-  
-```  
-  
-```  
-  
-Dim context As ResourceContext = ResourceManager.Current.DefaultContext  
-Dim list As New List(Of String)()  
-If CultureInfo.CurrentCulture.Name = "en-US" Then  
-   list.Add("fr-FR")  
-Else  
-   list.Add("en-US")  
-End If  
-context.Languages = list  
-block.Text += String.Format("{1}The current culture is {0}{1}",  
-                            CultureInfo.CurrentCulture, vbCrLf)  
-block.Text += String.Format("The current culture UI is {0}\n",  
-                            CultureInfo.CurrentUICulture)  
-' The example displays output Like the following if run on a system  
-' whose current culture Is English (United States):  
-'    The current culture Is fr-FR  
-'    The current UI culture Is fr-FR  
-  
-```  
-  
-   
+## The current culture and UWP apps  
+ In Universal Windows Platform (UWP) apps, the <xref:System.Globalization.CultureInfo.CurrentCulture%2A> and <xref:System.Globalization.CultureInfo.CurrentUICulture%2A> properties are read-write, just as they are in .NET Framework and .NET Core apps. However, in UWP apps, the two properties map to a single value, which is the first culture in the [Windows.ApplicationModel.Resources.Core.ResourceManager.DefaultContext.Languages](https://docs.microsoft.com/en-us/uwp/api/windows.applicationmodel.resources.core.resourcecontext#properties_) collection that can be used as a framework culture.  
+
+In .NET Framework and .NET Core apps, the current culture is a per-thread setting, and the <xref:System.Globalization.CultureInfo.CurrentCulture%2A> and <xref:System.Globalization.CultureInfo.CurrentUICulture%2A> properties reflect the culture and UI culture of the current thread only. In UWP apps, the current culture maps to the [Windows.ApplicationModel.Resources.Core.ResourceManager.DefaultContext.Languages](https://docs.microsoft.com/en-us/uwp/api/windows.applicationmodel.resources.core.resourcecontext#properties_) collection, which is a global setting. Setting the <xref:System.Globalization.CultureInfo.CurrentCulture%2A> or <xref:System.Globalization.CultureInfo.CurrentUICulture%2A> property changes the culture of the entire app; culture cannot be set on a per-thread basis.
```

Comment:
>collection [](start = 525, length = 10)

please remove the word collection as we map CurrentCulture and CurrentUUICulture to the first value in this collection